### PR TITLE
[FEATURE] Completed Hero Section 

### PR DESCRIPTION
**Pull request by adityayadav71**
1. Completed Issue #44
2. Added Hero.js file in src/components/HomePageFeatures/ folder
3. Replaced HomePageHeader component with New Hero component in pages/index.js
4. Added static assets for Hero in static/img/HeroImages
5. Updated tailwind.config.js with necessary extensions like bg image and inner box shadow for "View Projects" Button in Hero

### CODE REVIEW

File: src/components/HomepageFeatures/Hero.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+export default function Hero() {
+  return (
+    <section className="relative bg-slate-950 bg-hero-pattern bg-contain bg-no-repeat bg-bottom p-6 pb-0">
+      <div className="flex flex-col items-center justify-center mx-auto max-w-prose">
+        <p className="px-3 py-1 bg-slate-800 rounded-full z-10 lg:mb-3 mb-6">TCET Open Source</p>
```
Comment (JayeshVP24) [inline src/components/HomepageFeatures/Hero.js:7]: change `py-1` -> `pb-0.5` for more centered appearence
aligning text vertically is a bit of an hassle. this is accepted approach.

also add `text-gray-200`

File: src/components/HomepageFeatures/Hero.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <h1 className="lg:text-7xl sm:text-6xl text-5xl text-center font-semibold z-10 lg:mb-6 mb-9">The Best Open Source Club</h1>
+        <p className="sm:text-lg text-sm text-slate-400  text-center z-10 lg:mb-9 mb-12">
+          Cras massa lectus lacinia urna nec felis facilisi a <br />
+          interdum. Neque ipsum diam pharetra eu sed.
+        </p>
+        <button className="flex items-center justify-center gap-x-2 p-2 pl-4 md:mb-0 mb-3 border-b-2 border-indigo-700 shadow-1xl shadow-indigo-400 semibold rounded-full bg-indigo-600">
```
Comment (JayeshVP24) [inline src/components/HomepageFeatures/Hero.js:13]: add `font-semibold`

Comment (JayeshVP24) [reply src/components/HomepageFeatures/Hero.js:13]: add a hover effect of some type. up to you!

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]
Majority stuff is great. good work. just make the tiny changes

File: src/components/HomepageFeatures/Hero.js (hunk 2 above)
Comment (riyavishwa981) [inline src/components/HomepageFeatures/Hero.js:13]: add text-white here as well

**Review by riyavishwa981 — CHANGES_REQUESTED** [review]

**Comment by riyavishwa981** [comment]
Overall good work and well written code, just make some small changes as mentioned

**Review by riyavishwa981 — APPROVED** [review]

**Review by JayeshVP24 — APPROVED** [review]

**Pull request merged**